Core.Telemetry/ApplicationInsightsTelemetryClient.cs

```diff
@@ -68,6 +69,7 @@ public void TrackException(Exception exception, string message = null, IDictiona
             properties.Add("Message", message);
 
             this.telemetryClient.TrackException(exception, properties);
+            this.logger?.LogError(exception.Message, string.Join(Environment.NewLine, properties));
```

Comment:
>logger? [](http://example.com/codeflow?start=17&length=7)

Now that we rely on this in production (e.g., open telemetry) but there is a chance that this is null and ignored, bothers me. Is it too much change to ensure that this is provided?

Reply:
I have removed this. This will require some trivial refactoring in GitHub + ADO, but nothing major.